### PR TITLE
Create mission_goals.md

### PR DESCRIPTION
**Pull request by kimdhamilton**
Initial version of mission and goals for DID Method Standardization

### CODE REVIEW

**Review by thereisnogabe — APPROVED** [review]

File: mission_goals.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+4. Facilitating broader community coordination to support DID method standardization efforts across the ecosystem, including: 
+    1. Engaging with various standards groups, organizations, and individuals interested in DID method creation and standardization
+    2. Promoting knowledge sharing and best practices across the community 
+    3. Enabling discovery of efforts and collaborators
+
+## Objectives
+
+To achieve these goals, we will pursue the following objectives:
+
+### DIF Working Group
```
Comment (msporny) [inline mission_goals.md]:
```suggestion
### DIF Working Group with Joint Participation
```

I know it's happening under DIF, but can we further highlight that this is a joint initiative between multiple pre-SDOs?

File: mission_goals.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+A new “DID Method Standardization” working group (WG) at the [Decentralized Identity Foundation (DIF)](https://identity.foundation/) will perform the following activities in support of Goals 1 and 3.
```
Comment (msporny) [inline mission_goals.md]:
```suggestion
A new “DID Method Standardization” working group (WG) at the [Decentralized Identity Foundation (DIF)](https://identity.foundation/), in collaboration with a number of organizations (listed below), will perform the following activities in support of Goals 1 and 3.
```

**Review by msporny — CHANGES_REQUESTED** [review]
LGTM other than some minor adjustments highlighting that this is a multi-organization initiative.

**Comment by kimdhamilton** [comment]
> LGTM other than some minor adjustments highlighting that this is a multi-organization initiative.

Great suggestions @msporny, thank you!

**Pull request merged**